modules/schedule_spanner_backup/variables.tf

```diff
@@ -22,27 +22,31 @@ variable "project_id" {
 variable "instance_name" {
   description = "The name of the Spanner instance"
   type        = string
+  default     = "spanner-instance"
```

Comment:
Is is necessary to provide defaults for these variables vs leaving them as required?